wiki/Main_Page/en.md

```diff
@@ -98,9 +98,11 @@ Do your part: [How and what to help with](/wiki/Community/How_you_can_help!) •
 
 Social interaction is one of main things that fuel osu! players all around the world. Explore the activities and contests, get in touch with others, or help organise the next big event!
 
+Places: [Forum](/wiki/Community/Forum) • [Chat](/wiki/Community/Internet_Relay_Chat) • [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server)
```

Comment:
irc instead of chat console?

Reply:
well, it's technically chat. I wanted to make a small article on chat, but until then we'll have to use either

Reply:
i mean at least put in IRC as the link text instead lol

i suppose there could be more links in the client section now that i think about it

Reply:
I'll push another update as soon as there are release stream articles